codespell_lib/data/dictionary.txt

```diff
@@ -9286,6 +9286,7 @@ moreso->more, more so,
 morever->moreover
 morgage->mortgage
 morisette->morissette
+mormalized->normalized
```

Comment:
Can you add the s based variant please and the singulars (s and z).

Reply:
Done

Reply:
There's no mormalised or mormalises in here yet.